README.md

```diff
@@ -25,240 +25,237 @@ all while being non-Turing complete, strongly typed and [strongly normalizing][n
 (i.e.: reduces everything to a normal form, no matter how much abstraction you build),
 so saving you from the *"oh-noes-I-made-my-config-in-code-and-now-its-too-abstract"* nightmare.
 
-For a Dhall Tutorial, see the [readme of the project][dhall-lang],
+For a Dhall Tutorial, see [the website][dhall-website], or the [readme of the project][dhall-lang],
 or the [full tutorial][dhall-tutorial].
 
 ## Prerequisites
 
-**NOTE**: `dhall-kubernetes` requires at least version `1.20.1` of [the interpreter](https://github.com/dhall-lang/dhall-haskell)
-(version `5.0.0` of the language).
+**NOTE**: `dhall-kubernetes` requires at least version `1.21.0` of [the interpreter](https://github.com/dhall-lang/dhall-haskell)
+(version `6.0.0` of the language).
 
 You can install the latest version with the following:
 ```bash
-stack install dhall-1.20.1 dhall-json-1.2.6 --resolver=nightly-2019-01-17
+stack install dhall-1.21.0 dhall-json-1.2.7 --resolver=nightly-2019-04-01
 ```
 
-## Quickstart - main API
-
-We provide a simple API for the most common cases (For a list, see the [api](./api) folder).
+## Quickstart - a simple Deployment
 
 Let's say we'd like to configure a Deployment exposing an `nginx` webserver.
+
 In the following example, we:
-1. Define a `config` for our service, by merging a [default config][default-deployment]
-  (with the Dhall record-merge operator `//`) with a record with our parameters.
-2. In there we define the details of the Deployment we care about (note that we do the same
-  "merging with defaults" operation for our container as well, so we don't have to specify
-  all the parameters)
-3. We call the [`mkDeployment`][mkDeployment] function on our `config`
+1. Import the Kubernetes definitions as Dhall Types (the `types.dhall` file) from the local repo.
+   In your case you will want to replace the local path with a remote one, e.g.
+   `https://raw.githubusercontent.com/dhall-lang/dhall-kubernetes/0a4f0b87fbdd4b679853c81ff804bde7b44336cf/types.dhall`.  
```

Comment:
I know this is necessary until it's merged in, but you should probably replace the commit hash with a tag that will exist, e.g. `v2.0.0`